recipes/proj/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibaecConan(ConanFile):
+    name = "proj"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://proj.org"
+    description = "Cartographic Projections and Coordinate Transformations Library"
+    topics = ("conan", "dsp", "proj", "proj4", "projections", "gis", "geospatial")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+    }
+
+    generators = "cmake"
+    exports_sources = ["CMakeLists.txt", "sqlite_init.py", "patches/*"]
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("sqlite3/3.31.0")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["PROJ_TESTS"] = False
+        cmake.definitions["BUILD_LIBPROJ_SHARED"] = self.options.shared
+        cmake.configure(build_folder=self._build_subfolder)
+        return cmake
+
+    def build(self):
+        tools.patch(**self.conan_data["patches"][self.version])
+        tools.replace_in_file(os.path.join(self._source_subfolder, "src", "lib_proj.cmake"),
+                              "include_directories(${CMAKE_SOURCE_DIR}/include)",
+                              "include_directories(${PROJ4_SOURCE_DIR}/include)")
+        sqlite_check = "find_program(EXE_SQLITE3 sqlite3)\n" \
+                       "if(NOT EXE_SQLITE3)\n" \
+                       "  message(SEND_ERROR \"sqlite3 binary not found!\")\n" \
+                       "endif()\n"
+        tools.replace_in_file(os.path.join(self._source_subfolder, "CMakeLists.txt"), sqlite_check, "")
+
+        tools.replace_in_file(os.path.join(self._source_subfolder, "data", "CMakeLists.txt"),
+                              "COMMAND ${EXE_SQLITE3} -init ${ALL_SQL_IN} ${PROJ_DB} .quit",
+                              "COMMAND python ${CMAKE_SOURCE_DIR}/sqlite_init.py ${PROJ_DB} ${ALL_SQL_IN}")
```

Comment:
Maybe replace `python` with a combination of `find_package(PythonInterp)` and `PYTHON_EXECUTABLE`. ([docs](https://cmake.org/cmake/help/v3.0/module/FindPythonInterp.html))
For those cases where python has another filename.

Reply:
Was trying to keep it small, but this is easier as a patch anyway so I added this logic and pulled the find/replace out from conanfile.py